src/app/pages/things/opcua/opcua.component.ts

```diff
@@ -112,9 +119,10 @@ export class OpcuaComponent implements OnInit {
   opcuaNodes = [];
 
   browseServerURI = '';
-  browseNamespace = '';
-  browseIdentifier = '';
+  browseNamespace = '0';
+  browseIdentifier = 'i=84';
```

Comment:
Why is this 84 hard-coded and what is this exactly?

Reply:
The standard root OPC-UA server node

Reply:
Please put this in a comment then, because it is obacure